### ISSUE TITLE
Could you open source of the samples trained？

### ISSUE THREAD

**Issue by zjuwjf** [nanohop/sketch-to-react-native#34]
Awesome project！

Could you open source of the samples you trained ？

**Comment by chrisachard** [comment]
Thanks!  I could probably gather some together; I'd like to have (maybe a separate repo?) with the machine learning stuff (including samples to train), but for now I don't have a great way to do that.  Also, some of them are very large (file size), so there's not a great way to share them.

**Comment by peterpme** [comment]
Hey,

I would love to improve this. Is there a way you can jot down the process of running a neural network over this? 

Thanks!

**Comment by chrisachard** [comment]
Yes! It's been on my list to write this up, but I need to find time to do it.  Also - the way it's done now will probably not be the way it's done in the next version.  Here's a quick overview:

Now: it's done by just running sample images through a version of: https://codelabs.developers.google.com/codelabs/tensorflow-for-poets/#0  - but I don't think that's flexible enough.

Future: I want to write a new neural network (maybe with pytorch - I just used it for doing the most recent fast.ai stuff, and I liked it) to combine all the data I can from the Sketch file directly (example: where in a screen the component is located, along with what the component looks like).

To do that though, I first have to write this as a Sketch plugin, and not just as an SVG export parser.

If you have ideas about any of that - I'd be glad to talk!

**Comment by peterpme** [comment]
I have no idea how any of this stuff works (lol) but would love to help! Let me read this article. Thanks!

**Comment by peterpme** [comment]
This tutorial is cool!

So did you create a bunch of folders with different types of SVGs?
Button, Text, Layout, etc?

**Comment by chrisachard** [comment]
Yes, exactly - I did JPGs though

**Comment by peterpme** [comment]
Ah cool! So replacing `flower_photos` with `ui_photos` or something and folders for different types of UI that are JPGs.

I would love to write up some guide for others who are interested in making this plugin better!
